### ISSUE TITLE
Arxiv_Issue

### ISSUE THREAD

**Issue by AkiraTOSEI** [AkiraTOSEI/ML_papers#5]
# TL;DR
 They proposed a spatially conditional activation functi0on, FReLU, which is better than ReLU and PReLU because it can determine whether or not each pixel is active or not, taking into account the surrounding information. They confirmed the improved accuracy in image recognition, object detection, and region segmentation.
![image](https://user-images.githubusercontent.com/24524018/89172643-ec681200-d5bd-11ea-8a5a-69bb18f85e53.png)
![image](https://user-images.githubusercontent.com/24524018/89172655-effb9900-d5bd-11ea-80ab-ac9123235c1f.png)

# Why it matters:

# Paper URL
https://arxiv.org/abs/2007.11824

# Submission Dates(yyyy/mm/dd)
2020/07/23

# Authors and institutions
Ningning Ma, Xiangyu Zhang, Jian Sun 
- Hong Kong University of Science and Technology MEGVII Technology 
# Methods

# Results

# Comments
ECCV2020
